internal/controller/driver_controller.go

```diff
@@ -1067,6 +1209,13 @@ func (r *driverReconcile) reconcileNodePluginDeamonSet() error {
 								utils.OidcTokenVolume,
 							)
 						}
+						if logRotationEnabled {
+							volumes = append(
+								volumes,
+								utils.LogsDirVolume(logHostPath, daemonSet.Name),
```

Comment:
what will happen for the case where we run two drivers in different namespaces that can get the same daemonset Name?

Reply:
daemonset and deployment Name is genrated using a generate function

`return fmt.Sprintf("%s-%s", r.driver.Name, suffix)`

The generate function appends the `drivername` as a prefix so no two `daemonset Name` can ever b same if its a differnt driver